src/coreclr/inc/corhlprpriv.cpp

```diff
@@ -109,6 +109,23 @@ HRESULT _CountBytesOfOneArg(
     }
     switch (ulElementType)
     {
+        case ELEMENT_TYPE_GENERICINST:
+            CHECK_REMAINDER;
```

Comment:
Should `CHECK_REMAINDER` be in front of `CorSigUncompressData` instead?

Reply:
And there should be `cb = cbTotalMax - cbTotal;` in front of `_CountBytesOfOneArg` calls.

Reply:
I see, will fix that.